ScnSoft-Education/LeetCodeTraining.Tests/RemoveDuplicatesFromSortedArrayTest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using LeetCodeTraining.RemoveDuplicatesFromSortedArray;
+
+namespace LeetCodeTraining.Tests
+{
+    public class RemoveDuplicatesFromSortedArraySolutionTest
+    {
+        [Theory]
+        [InlineData(new int[] { 1, 1, 2 }, 2)]
+        [InlineData(new int[] { 0, 0, 1, 1, 1, 2, 2, 3, 3, 4 }, 5)]
+        public void RemoveDuplicatesFromSortedArray(int[] nums, int expectedResult)
+        {
+            //Act
+            int result = RemoveDuplicatesFromSortedArraySolution.RemoveDuplicates(nums);
```

Comment:
Please add tests on checking that first k values of nums are expected unique values.
![image](https://github.com/ArinaSiselina/ScnSoft-Education/assets/101803780/f54d065a-d6ec-40e5-8657-51c3a1b5ce53)